File: app/src/main/java/com/github/starter/Application.java
package com.github.starter;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.r2dbc.R2dbcAutoConfiguration;

@SpringBootApplication(exclude = {R2dbcAutoConfiguration.class})
class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class);

    }
}

File: app/src/main/kotlin/com/github/starter/Application.kt
package com.github.starter;

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.r2dbc.R2dbcAutoConfiguration

@SpringBootApplication(exclude = arrayOf(R2dbcAutoConfiguration::class))
open class Application

fun main() {
    SpringApplication.run(Application::class.java)
}

